core/templates/dev/head/player/PlayerServices.js

```diff
@@ -357,10 +378,30 @@ oppia.factory('oppiaPlayerService', [
     getGadgetPanelsContents: function() {
       return angular.copy(_exploration.skin_customizations.panels_contents);
     },
+    getInteractionThumbnailSrc: function(stateName) {
+      // TODO(sll): unify this with the 'choose interaction' modal in
+      // state_editor_interaction.html.
+      var interactionId = _exploration.states[stateName].interaction.id;
+      if (!interactionId) {
+        return '';
+      } else {
+        return (
+          '/extensions/interactions/' + interactionId + '/static/' +
+          interactionId + '.png');
+      }
+    },
+    getInteractionInstructions: function(stateName) {
+      var interactionId = _exploration.states[stateName].interaction.id;
+      if (!interactionId) {
+        return '';
+      } else {
+        return INTERACTION_SPECS[interactionId].instructions;
+      }
+    },
     isInteractionInline: function(stateName) {
       var interactionId = _exploration.states[stateName].interaction.id;
       return (
-        interactionId &&
+        !interactionId ||
```

Comment:
This seems like a conspicuous change to me. Could you add a comment explaining why a non-interaction should be inline?